book/src/validator-monitoring.md

```diff
@@ -96,4 +96,86 @@ Jan 18 11:21:09.808 INFO Attestation included in block           validator: 1, s
+
+The attestation simulation prints the following logs when simulating an attestation:
+
+```
+DEBG Simulating unagg. attestation production, service: beacon, module: beacon_chain::attestation_simulator:39
+DEBG Produce unagg. attestation, attestation_target: 0x59fc…1a67, attestation_source: 0xc4c5…d414, service: beacon, module: beacon_chain::attestation_simulator:87
+```
+
+When the simulated attestation has completed, it prints a log that specifies if the head, target and source votes are hit.  An example of a log when all head, target and source are hit:
+
+```
+DEBG Simulated attestation evaluated, head_hit: true, target_hit: true, source_hit: true, attestation_slot: Slot(1132616), attestation_head: 0x61367335c30b0f114582fe298724b75b56ae9372bdc6e7ce5d735db68efbdd5f, attestation_target: 0xaab25a6d01748cf4528e952666558317b35874074632550c37d935ca2ec63c23, attestation_source: 0x13ccbf8978896c43027013972427ee7ce02b2bb9b898dbb264b870df9288c1e7, service: val_mon, service: beacon, module: beacon_chain::validator_monitor:2051
+```
+
+An example of a log when the head is missed:
+```
+DEBG Simulated attestation evaluated, head_hit: false, target_hit: true, source_hit: true, attestation_slot: Slot(1132623), attestation_head: 0x1c0e53c6ace8d0ff57f4a963e4460fe1c030b37bf1c76f19e40928dc2e214c59, attestation_target: 0xaab25a6d01748cf4528e952666558317b35874074632550c37d935ca2ec63c23, attestation_source: 0x13ccbf8978896c43027013972427ee7ce02b2bb9b898dbb264b870df9288c1e7, service: val_mon, service: beacon, module: beacon_chain::validator_monitor:2051
+```
+
+
+With `--metrics` enabled on the beacon node, the following metrics will be recorded:
+
+```
+validator_monitor_attestation_simulator_head_attester_hit_total
+validator_monitor_attestation_simulator_head_attester_miss_total
+validator_monitor_attestation_simulator_target_attester_hit_total
+validator_monitor_attestation_simulator_target_attester_miss_total
+validator_monitor_attestation_simulator_source_attester_hit_total
+validator_monitor_attestation_simulator_source_attester_miss_total
+```
+
+We can query the metrics for `validator_monitor_attestation_simulator_head_attester_hit_total`:
+
+```bash
+ curl -s "http://localhost:5054/metrics" | grep validator_monitor_attestation_simulator_head_attester_hit_total
+ ```
+
+ An example of response is:
+
+```
+# HELP validator_monitor_attestation_simulator_head_attester_hit_total Incremented if a validator is flagged as a previous slot head attester during per slot processing
+# TYPE validator_monitor_attestation_simulator_head_attester_hit_total counter
+validator_monitor_attestation_simulator_head_attester_hit_total 40666
+```
+
+Similarly, to query the metrics for `validator_monitor_attestation_simulator_head_attester_miss_total`:
+
+```bash
+ curl -s "http://localhost:5054/metrics" | grep validator_monitor_attestation_simulator_head_attester_miss_total
```

Comment:
I don't think users should be querying prometheus metrics manually. Instead this metrics should be integrated in a dashboard that we can link and tell users to import into their grafana instances.

Reply:
Make sense. I move the Grafana sentence upwards and remove the manual queries here: 

[2a1edb2](https://github.com/sigp/lighthouse/pull/5364/commits/2a1edb2e7c6679f8dde42a2345e6165f6c223fe2)